addons/mail/models/mail_channel.py

```diff
@@ -605,7 +605,16 @@ def message_post(self, *, message_type='notification', **kwargs):
         if moderation_status == 'rejected':
             return self.env['mail.message']
 
-        self.filtered(lambda channel: channel.is_chat).mapped('channel_last_seen_partner_ids').sudo().write({'is_pinned': True})
+        if self.is_chat:
+            self.mapped('channel_last_seen_partner_ids').sudo().write({'is_pinned': True})
+            partner_ids = self.channel_partner_ids.ids
+            self.env['mail.channel.partner'].search([
+                ('partner_id', 'in', partner_ids),
+                ('channel_id', '=', self.id),
+            ]).write({
+                'last_activity_time': fields.Datetime.now()
+            })
+            self._broadcast(partner_ids)
```

Comment:
There will already be another bus notification as part of message post (call to `_notify_thread`), can't we combine both?

At least it is strange to send here the `channel_info` on the bus before the message is actually posted, because it means other values (such as last message, unread counter, ... will not be sent with their new value at this time).

Reply:
the notification is now sent in `_notify_thread`